test/unit/app/models/contact_test.rb

```diff
@@ -292,4 +292,21 @@ class ContactTest < ActiveSupport::TestCase
       office.contact.destroy!
     end
   end
+
+  test "when associated with a WorldwideOrganisationPage should be valid when translated into a language that the worldwide organisation has" do
+    worldwide_organisation = create(:editionable_worldwide_organisation, translated_into: %i[de es fr])
+    office = create(:worldwide_office, edition: worldwide_organisation, worldwide_organisation: nil)
+    contact = create(:contact, contactable: office, translated_into: %i[fr])
+
+    assert contact.valid?
+  end
+
+  test "when associated with a WorldwideOrganisationPage should not be valid when translated into a language that the worldwide organisation does not have" do
```

Comment:
`when associated with a WorldwideOffice should not be valid when translated into a language that the worldwide office's associated organisation does not have`?

Reply:
Have updated.